.gitignore

```diff
@@ -1,4 +1,2 @@
 target/
-
-# api keys
-.env
+*keys.js
```

Comment:
Maybe using .env to hide the api key here is better cause I think we can still see keys.js if we do inspect page... but not sure about this so make your own judgement on it. :)

Reply:
Yes, I have just kept it until we figure out a way to do the secrets, which is till tomorrow. I will be removing that to .env after we do that tomorrow.